.golangci.json

```diff
@@ -26,6 +25,10 @@
         "path": "foldr_test\\.go",
         "text": "SA4000:",
         "severity": "warning"
+      },
+      {
+        "path": "(dawgs/util/size/(.+)|dawgs/drivers/pg/statements.go)",
```

Comment:
Sizing has some unused fields, but that's doing terrible things to derive sizes, so best to not run unused on it. pg/statements.go appears to have some pretty important query statements without usages, but given how new things are, it seems ok to ignore these warnings for now

Reply:
For statements, that makes sense. I wanted to be extra sensitive to removing anything from the size package just due to not entirely knowing if it could affect size calculations. I can show you what the linter is flagging just to validate